### ISSUE TITLE
Clients: Provide easy method to retrieve a Thing's Root Type

### ISSUE THREAD

**Issue by jmsfltchr** [graknlabs/grakn#6007]
## Problem to Solve

Previously it was possible to access the base Type to determine whether it was an Entity/Relation/Attribute quite trivially. In the Grakn 2.0 alpha clients this requires 3 if checks instead.

Previously in Python we could write:
```
id = grakn_thing.id
type_label = grakn_thing.type().label()
base_type_label = grakn_thing.as_remote(tx).base_type.replace('_TYPE', '').lower()
```

## Current Workaround

Now we need to write:
```
    id = grakn_thing.get_iid()
    type_label = grakn_thing.get_type().get_label()
    if grakn_thing.isEntity():
        base_type_label = "entity"
    elif grakn_thing.isRelation():
        base_type_label = "relation"
    elif grakn_thing.isAttribute():
        base_type_label = "attribute"
    else:
        raise RuntimeError("Unexpected Concept")
```

## Proposed Solution

Perhaps we should have a more streamlined way to determine this. I expected to be able to say something like:
```
base_type_label = grakn_thing.get_type().root_type()
```

**Comment by haikalpribadi** [comment]
Is this blocking you from building something right now, @jmsfltchr ? If so, move it back to "Grakn 2.0 Production". Otherwise, I'm moving this to backlog.

**Comment by jmsfltchr** [comment]
> Is this blocking you from building something right now, @jmsfltchr ? If so, move it back to "Grakn 2.0 Production". Otherwise, I'm moving this to backlog.

No it doesn't block anything, simply means more lines of code than before to find the root type. Moving to backlog.

**Comment by haikalpribadi** [comment]
Where are you having to write this "more lines of code", @jmsfltchr ?

**Comment by jmsfltchr** [comment]
I thought that was clear in the description where I compare previously with now! We now need to do 3 `if` checks rather than directly asking for the root type as we did before. If instead you mean where as in which repo, there's just one occurrence in KGLIB.

**Comment by haikalpribadi** [comment]
I can tell it's using client-python, but I couldn't tell it's for KGLIB - I did have a hunch, though. Aite, then it's okay for now. But we can pick this issue up soon after production.

**Comment by jmsfltchr** [comment]
Yes totally fine for now, but I'd say an awkward UX for the uninitiated.
